gradoop-examples/src/main/java/org/gradoop/examples/rollup/TelephoneCallGraph.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.examples.rollup;
+
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.apache.flink.api.java.ExecutionEnvironment;
+import org.gradoop.flink.io.impl.gdl.GDLDataSink;
+import org.gradoop.flink.model.impl.epgm.GraphCollection;
+import org.gradoop.flink.model.impl.epgm.LogicalGraph;
+import org.gradoop.flink.model.impl.operators.grouping.Grouping;
+import org.gradoop.flink.model.impl.operators.grouping.functions.aggregation.CountAggregator;
+import org.gradoop.flink.util.FlinkAsciiGraphLoader;
+import org.gradoop.flink.util.GradoopFlinkConfig;
+
+/**
+ * A self contained example on how to use the rollUp operator on Gradoop's LogicalGraph class.
+ *
+ * The example uses our own telephone call example graph.
+ */
+public class TelephoneCallGraph {
+
+  /**
+   * Path to the example data graph
+   */
+  private static final String EXAMPLE_DATA_FILE =
+    TelephoneCallGraph.class.getResource("/data/gdl/tc.gdl").getFile();
+
+  /**
+   * Runs the program on the example data graph.
+   *
+   * The example provides an overview over the usage of the different rollUp methods.
+   * It showcases the application of rollUp functions that are already provided by
+   * Gradoop. Documentation for all available rollUp functions can be found in the projects wiki.
+   *
+   * @see <a href="https://github.com/dbs-leipzig/gradoop/wiki/Unary-Logical-Graph-Operators">
```

Comment:
Wrong usage of `@see` annotation. @see https://www.oracle.com/technetwork/articles/java/index-137868.html